src/init.ts

```diff
@@ -98,13 +98,13 @@ export async function init(signerService: SignerService) {
     );
 
     async function connectToDidRegistry(
-      ipfsStore?: string
+      ipfsConfig: IpfsConfig
```

Comment:
Is there a reason why this should stay optional? Should there be a default config set in ICL in case it is not passed in?